contrib/conf/layer.conf

```diff
@@ -13,3 +13,13 @@ LAYERVERSION_tegra-contrib = "1"
 LAYERDEPENDS_tegra-contrib = "core"
 
 LAYERSERIES_COMPAT_tegra-contrib = "warrior"
+
+# -fmacro-prefix-map isn't supported in GCC 7
```

Comment:
Rather than adding these settings here, could you create another config file for them, under, say `contrib/conf/include`?  I'd rather the flags additions be opt-in separately from adding the layer (which could expand to include other optional stuff).